src/views/stadiumview.hpp

```diff
@@ -10,17 +10,74 @@ namespace Ui {
 class StadiumView;
 }
 
+/**
+ *@class StadiumView class
+ *
+ * This class is used for displaying the information in the database
+ * in various different ways. It can display all teams and stadiums,
+ * National/American league teams only, etc.
+ *
+ * This class keeps track of what is being displayed as a call
+ * to resetUi will make sure that the tree widget gets repopulated,
+ * but to whatever the last button clicked was
+ *
+ * Note:
+ * This view assumes that the database is initialized with data for it to read out of.
+ */
 class StadiumView : public View
 {
     Q_OBJECT
 
 public:
+    /* Constructor*/
     explicit StadiumView(QWidget* parent = nullptr);
+
+    /* Destructor*/
     ~StadiumView() override;
+
+    /* Resets */
     void resetView() override;
     void resetUi() override;
 
+    /* Find Max/Min Distance Teams */
+    std::vector<std::pair<Team,Stadium>> maxDistanceTeams(std::vector<std::pair<Team,Stadium>>);
+    std::vector<std::pair<Team,Stadium>> minDistanceTeams(std::vector<std::pair<Team,Stadium>>);
+
+private slots:
+    void onStadiumClicked(int);
```

Comment:
This function name has no meaning at all in terms of what it does. It looks more like a signal than a slot/function. Something like `displayTeamInfo` would've worked (don't use that name unless you want to, just an example).